### PR TITLE
example: add arithmetic mean example from rosetta code

### PR DESCRIPTION
**Pull request by Morriar**
Signed-off-by: Alexandre Terrasa <alexandre@moz-code.org>

### CODE REVIEW

**Comment by Morriar** [comment]
ok to test

**Comment by jpages** [comment]
+1

File: examples/rosettacode/arithmetic_mean.nit [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+#
+# This file is part of NIT ( http://www.nitlanguage.org ).
+# This program is public domain
+
+# Task: Arithmetic mean
+# SEE: <http://rosettacode.org/wiki/Averages/Arithmetic_mean>
+module arithmetic_mean
+
+# Computes the meam or the `array` values.
+fun mean(array: Array[Int]): Float do
```
Comment (privat) [inline examples/rosettacode/arithmetic_mean.nit]: I thing you should pass an Array[Float] to simplify the code and be more type consistant

**Comment by Morriar** [comment]
Reroll with floats

**Comment by privat** [comment]
+1

**Pull request merged**